scene/resources/style_box.cpp

```diff
@@ -552,7 +551,7 @@ inline void set_inner_corner_radius(const Rect2 style_rect, const Rect2 inner_re
 	inner_corner_radius[0] = MAX(corner_radius[0] - rad, 0);
 
 	//tr
-	rad = MIN(border_top, border_bottom);
+	rad = MIN(border_top, border_right);
```

Comment:
This is a little fix for another glitch that could cause the top right rounded corner to be drawn with the wrong size in some cases.

Reply:
ups that was obviously a mistake.